.ci/atime/tests.R

```diff
@@ -66,61 +66,60 @@ pkg.edit.fun = function(old.Package, new.Package, sha, new.pkg.path) {
 #
 # Optional parameters that may be useful to configure tests:
 # - times: Number of times each expression is evaluated (default is 10).
-# - seconds.limit: The maximum median timing (in seconds) of an expression. No timings for larger N are computed past that threshold.
-# - sha.vec: Named character vector or a list of vectors that specify data.table-specific commit SHAs for testing across those different git commit versions.
-#            For historical regressions, use 'Before', 'Regression', and 'Fixed' (otherwise something like 'Slow' or 'Fast' ideally).
+# - seconds.limit: The maximum median timing (in seconds) of an expression. No timings for larger N are computed past that threshold. Default of 0.01 seconds should be sufficient for most tests.
+# - Historical references (short version name = commit SHA1).
+#   For historical regressions, use version names 'Before', 'Regression', and 'Fixed'
+#   When there was no regression, use 'Slow' and 'Fast' 
 # @note Please check https://github.com/tdhock/atime/blob/main/vignettes/data.table.Rmd for more information.
 # nolint start: undesirable_operator_linter. ':::' needed+appropriate here.
-test.list <- list(
+test.list <- atime::atime_test_list(
   # Performance regression discussed in: https://github.com/Rdatatable/data.table/issues/4311
   # Fixed in: https://github.com/Rdatatable/data.table/pull/4440
-  "shallow regression fixed in #4440" = list(
-    pkg.edit.fun = pkg.edit.fun,
+  pkg.edit.fun = pkg.edit.fun,
```

Comment:
using atime_test_list, pkg.edit.fun now can now be defined once here, and then shared among all tests

Reply:
Looks like it isn't being detected:
```sh
Error in atime::atime_pkg(Sys.getenv("GITHUB_WORKSPACE"), tests.dir = ".ci") : 
  object 'pkg.edit.fun' not found
Calls: <Anonymous> -> do.call -> <Anonymous>
Execution halted
```
(I tested it on my fork with these changes and got the same too)

Reply:
Seems to be an issue with the scope (defining the function inside the `atime::atime_test_list` call fixed it for me on my fork)